aspnetcore/release-notes/aspnetcore-10/includes/doc-provider-in-di.md

```diff
@@ -1,6 +1,8 @@
 ## Support for IOpenApiDocumentProvider in the DI container.
-<!-- https://github.com/dotnet/aspnetcore/pull/61463 -->
-ASP.NET has added support for `IOpenApiDocumentProvider` in the DI container.
-This allows you to inject the `IOpenApiDocumentProvider` into your application and use it to access the OpenAPI document.
-This is useful for scenarios where you need to access the OpenAPI document outside of the context of an HTTP request,
-such as in a background service or a custom middleware.
+
+ASP.NET Core in .NET 10 supports [IOpenApiDocumentProvider](https://source.dot.net/#Microsoft.AspNetCore.OpenApi/Services/IOpenApiDocumentProvider.cs) in the dependency injection (DI) container. Developers can inject `IOpenApiDocumentProvider` into their apps and use it to access the OpenAPI document. This approach is useful for accessing OpenAPI documents outside the context of HTTP requests, such as in background services or custom middleware.
+
+Previously, running application startup logic without launching an HTTP server could be done by using `HostFactoryResolver` with a no-op `IServer` implementation. The new feature simplifies this process by providing a streamlined API inspired by Aspire's <xref:Aspire.Hosting.Publishing.IDistributedApplicationPublisher>.
```

Comment:
I think this paragraph is describing functionality that was not actually implemented because of concerns raised in the API review. The `IOpenApiDocumentProvider` interface does not change the way OpenAPI documents are generated at build time. So I recommend simply removing this paragraph.  @captainsafia can you pls confirm?
```suggestion
```